tests/Aspire.Playground.Tests/AppHostTests.cs

```diff
@@ -51,19 +53,29 @@ public async Task TestEndpointsReturnOk(TestEndpoints testEndpoints)
 
         await app.StartAsync();
 
-        var applicationModel = app.Services.GetRequiredService<DistributedApplicationModel>();
-        var resourceNotificationService = app.Services.GetRequiredService<ResourceNotificationService>();
+        if (testEndpoints.WaitForTexts != null)
+        {
+            // If specific ready to start texts are available use it
+            var tasks = testEndpoints.WaitForTexts.Select(x => app.WaitForTextAsync(log => new Regex(x.Pattern).IsMatch(log), x.ResourceName));
+            await Task.WhenAll(tasks).WaitAsync(TimeSpan.FromMinutes(2));
```

Comment:
not for this PR, but it might become useful to have a custom timeout too for the extra slow resources :D